src/prefabs/backOffice.tsx

```diff
@@ -5631,6 +5632,87 @@ const beforeCreate = ({
     },
   });
 
+  const searchPropDisabledKinds = createBlacklist([
```

Comment:
Google doc and login token should probably not be whitelisted

Reply:
I don't think it is possible to use the Object property in the beforeCreate as well